### ISSUE TITLE
Home Page - WCAG compliance

### ISSUE THREAD

**Issue by steffipravasi** [bcgov/performance#1016]

**Comment by steffipravasi** [comment]
Ticket updated with the Accessibility elements

**Comment by steffipravasi** [comment]
Issue 20, 21, 29 and 30 has been removed from this ticket and added separately

**Comment by jizhaogit** [comment]
@telusdcinco Hi Dennis, please help the code review. thanks!

**Comment by jizhaogit** [comment]
deployment notice:
just code migrate

**Comment by steffipravasi** [comment]
@jizhaogit Can you please confirm if this ticket has been deployed to test env. I don't see the fixes yet.

**Comment by steffipravasi** [comment]
@jizhaogit Issues we'll need to take a look again – 1, 5, 7, 9, 12, 13, 15, 18, 19, 24, 25, 26, 28

**Comment by steffipravasi** [comment]
Issues on Apr 24 – 5, 7, 15, 19, 24, 25  
Need to discuss with Ji before assigning back

**Comment by steffipravasi** [comment]
Ticket discussed with Ji. Assigning back for fixes.

**Comment by jizhaogit** [comment]
@steffipravasi the issues should be addressed. thank you!

**Comment by steffipravasi** [comment]
@jizhaogit There is still one item left to be fixed in the list
Issues on Apr 25 - Item no 7

**Comment by steffipravasi** [comment]
All accessibility issues have been addressed.
